decidim-elections/app/cells/decidim/elections/election_m_cell.rb

```diff
@@ -17,6 +17,14 @@ def title
         present(model).title
       end
 
+      def has_image?
+        model.attachments.first.present? && model.attachments.first.file.content_type.start_with?("image")
+      end
```

Comment:
Here you could use the `:photos` method from [`HasAttachments`](https://github.com/decidim/decidim/blob/f92b6a5b18d53c0f3844f39b8db019eb07c74c6d/decidim-core/lib/decidim/has_attachments.rb#L18-L23) concern and the `:photo?` method from the [`Attachment`](https://github.com/decidim/decidim/blob/f92b6a5b18d53c0f3844f39b8db019eb07c74c6d/decidim-core/app/models/decidim/attachment.rb#L16-L21) model:

```suggestion
      def has_image?
        model.attachments.photos.present? && model.attachments.first.photo?
      end
```

Reply:
Awesome! I'll change it.